packages/synapse-react-client/src/components/GenericCard/PortalDOI/PortalDOI.tsx

```diff
@@ -56,8 +58,8 @@ function PortalDOI(props: PortalDOIProps) {
       >
         {doi && (
           <>
-            <Link href={doi.doiUrl} target={'_blank'}>
-              {doi.doiUrl}
+            <Link href={doiOrgUrl} target={'_blank'}>
+              {doiOrgUrl}
```

Comment:
`doi.doiUrl` was the Synapse backend URL used to get the URL. The design calls for the `https://doi.org` URL, fixed here.